src/api/public/apidocs/paths/build_project_name_repository_name_architecture_name_package_name_file_name_view_fileinfo.yaml

```
@@ -1,6 +1,6 @@
 get:
   summary: This endpoint returns details about an specific artifact
-  description: |
+  description: 
     Given a project, repository, architecture and package, retrieve the given artifact's details.
```

```suggestion
  description: Given a project, repository, architecture and package, retrieve the given artifact's details.
```
This was not moved up...